geode-core/src/test/java/org/apache/geode/management/internal/beans/stats/VMStatsMonitorTest.java

```diff
@@ -107,7 +89,7 @@ public void calculateCpuUsageShouldCorrectlyCalculateTheCpuUsed() {
   @Test
   public void refreshStatsShouldUpdateCpuUsage() {
     ZonedDateTime now = ZonedDateTime.now();
-    when(MBeanJMXAdapter.isAttributeAvailable(anyString(), anyString())).thenReturn(true);
+    vmStatsMonitor = new VMStatsMonitor(testName.getMethodName(), true);
```

Comment:
You are creating two VMStatsMonitor objects (line 92 and 93) where I think you meant to create one.  You can combine the two into just one line like:
```
vmStatsMonitor = spy(new VMStatsMonitor(testName.getMethodName(), true));
```

Reply:
fixed.